Containers/mastercontainer/start.sh

```diff
@@ -15,9 +15,20 @@ elif ! mountpoint -q "/mnt/docker-aio-config"; then
     exit 1
 elif ! sudo -u www-data test -r /var/run/docker.sock; then
     echo "Trying to fix docker.sock permissions internally..."
-    GROUP="$(stat -c '%g' /var/run/docker.sock)"
-    groupadd -g "$GROUP" docker && \
-    usermod -aG docker www-data
+
+    # check if a group with the same group id of /var/run/docker.socket already exists in the container
+    if grep -q $(stat -c '%g' /var/run/docker.sock) /etc/group; then
```

Comment:
Shouldn't it be 
```suggestion
    if grep -q "^$(stat -c '%G' /var/run/docker.sock):" /etc/group; then
```
?
Also storing this in a varialbe makes probably sense since we need it at two places...

Reply:
Also storing stat -c '%G' /var/run/docker.sock in a varialbe makes probably sense since we need it at two places...

Reply:
Are you sure we need that? It looks to me like everything runs with root permissions in that watchtower container. So we shouldn't have to deal with permissions.

Reply:
if the root user doesn't have problems with different permissions set then we might consider actually removing the whole logic: https://github.com/nextcloud/all-in-one/blob/a1f00db3e3038c2d5380881590732bd75cad7cd3/Containers/watchtower/start.sh#L7-L16
But I didn't test it out.

Reply:
@pestotoast do you mind resolving https://github.com/nextcloud/all-in-one/pull/50#discussion_r762449071 and https://github.com/nextcloud/all-in-one/pull/50#discussion_r762449499 ? Otherwise I'd take this over :)

Reply:
Regarding https://github.com/nextcloud/all-in-one/pull/50#discussion_r762449071: I was using `%g` instead of `%G` to be able to check for the group id instead of the group name.
I added some refactoring  regarding https://github.com/nextcloud/all-in-one/pull/50#discussion_r762449499

Reply:
Yes, but this might lead to problem since you only check if this gid is somewhere in the file.
This is not fail-proof enough, imho.